plugins/push/frontend/public/javascripts/countly.views.js

```diff
@@ -1677,9 +1686,63 @@
             getRemainingStackBar: function(value) {
                 return {data: [100 - value], itemStyle: {color: "#E2E4E8"}, silent: true};
             },
-
             onDrawerClose: function() {
                 this.$store.dispatch('countlyPushNotification/details/onSetIsDrawerOpen', false);
+            },
+            onGoToSent: function() {
+                var queryData = {message: {$in: ["609526bd23c8ba7ef0b1e099"]}};
```

Comment:
What is this harcoded thing?

Reply:
I fixed it for both cases. Forgot to replace the actual one.